### DISCUSSION TITLE
Breadcrumbs and client side navigation

### DISCUSSION THREAD

**Discussion by pchaganti** [cloudscape-design/components#1021] [Q&A] [answered]
Are there any samples or info on how to use client side nav with breadcrumbs. Trying to avoid the page reload with the Link component. I have an edit page and once the edit is saved, the user comes back to the original page. Since i am using the Link the page is reloading. I saw an earlier post about nextjs, but i am just using plain react.

Any pointers or help really appreciated.

thanks!

**Comment by Al-Dani** [comment, marked as answer]
As our [documentation for Link component](https://cloudscape.design/components/link/?tabId=api#events) recommends, you can prevent default behaviour for the event and apply desired navigation yourself. Same works for breadcrumbs
